staging/src/k8s.io/legacy-cloud-providers/azure/retry/azure_retry.go

```diff
@@ -138,6 +138,7 @@ func (b *Backoff) Step() time.Duration {
 // This allows clients to avoid converging on periodic behavior. If maxFactor
 // is 0.0, a suggested default value will be chosen.
 func jitter(duration time.Duration, maxFactor float64) time.Duration {
```

Comment:
We definitely don't want to do this in non-test code

we could either:

- improve the test to not require deterministic jittering (which is weird to expect to begin with)
- seed this only in the unit test
- use a private instance we seed, not the global